assets/js/main.js

```diff
@@ -151,11 +151,30 @@ const form = document.getElementById('contact-form');
 form.addEventListener('submit', (e) => {
   e.preventDefault();
   const email = document.getElementById('email').value;
+  const name = document.getElementById('name').value;
+  const message = document.getElementById('message').value;
+  const formData = {
+    email,
+    name,
+    message,
+  };
```

Comment:
Nice job 👍 persisting the data to local storage and getting it back on page load. But as required in the [project description](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md#project-requirements) the form data should have to be saved when the user changes the content of any `input field`, the data is saved to the local storage. Currently, the data will be saved when the user submits the form. Kindly, add a function to save the form data to `local storage` after the user leaves that particular field. For example, after entering the name and when typing the email the data should be saved in the local storage the same for email and for the message.